package.nls.de.json

```
@@ -17,28 +17,28 @@
-    "mql_tools.commands.InsImport.title": "DLL/EX5 als #import einfügen",
-    "mql_tools.commands.InsTime.title": "Datum und Uhrzeit als JJJJ.MM.TT hh:mm:ss einfügen",
-    "mql_tools.commands.InsIcon.title": "Symbol als #property einfügen",
-    "mql_tools.commands.openInME.title": "In 'MetaEditor' öffnen",
-    "mql_tools.commands.openTradingTerminal.title": "Handelsterminal öffnen",
-    "mql_tools.commands.createCommentary.title": "Funktionskommentar erstellen",
-    "mql_tools.commands.reportNoisyDiagnostic.title": "Störende clangd-Diagnose melden",
-    "mql_tools.commands.runBacktest.title": "Backtest ausführen",
-    "mql_tools.commands.startDebugging.title": "MQL-Debug-Sitzung starten",
-    "mql_tools.commands.stopDebugging.title": "MQL-Debug-Sitzung beenden",
-    "mql_tools.commands.showStartupPage.title": "Willkommensseite anzeigen"
+    "mql_tools.configuration.AutoVersionBump.description": "Automatically bump the minor version in #property version before each compile.","mql_tools.configuration.VersionConstantNames.description": "Names of const string variables to auto-bump on compile (e.g. EA_VERSION).","mql_tools.commands.compileFile.title": "MQL-Datei kompilieren",
```

Biome (2.4.15)

[error] The key mql_tools.configuration.AutoVersionBump.description was already declared.

(lint/suspicious/noDuplicateObjectKeys)